admin/client/src/components/Tabs/Tabs.js

```diff
@@ -0,0 +1,113 @@
+import React from 'react';
+import SilverStripeComponent from 'lib/SilverStripeComponent';
+import { Tab, Nav, NavItem } from 'react-bootstrap-ss';
+
+class Tabs extends SilverStripeComponent {
+  /**
+   * Determines a default tab to be opened and validates the given default tab.
+   * Replaces the given default tab if it is invalid with a valid tab.
+   *
+   * @returns {string|undefined}
+   */
+  getDefaultActiveKey() {
+    let active;
+
+    if (typeof this.props.defaultActive === 'string') {
+      const activeChild = React.Children.toArray(this.props.children)
+        .find((child) => child.props.name === this.props.defaultActive);
+
+      if (activeChild) {
+        active = activeChild.props.name
+      }
+    }
+
+    if (typeof active !== 'string') {
+      React.Children.forEach(this.props.children, (child) => {
+        if (active === null) {
```

Comment:
This was `active === undefined` for me (merge error?). `active` is always defined. I've changed to `typeof active === 'string'`